knapsack-gosdk/go.mod

```diff
@@ -1,8 +1,14 @@
 module example.com/your_project/knapsack-gosdk
 
-go 1.19
+go 1.21
 
-require github.com/nextmv-io/sdk v1.2.0
+toolchain go1.22.0
```

Comment:
What does this do and why does it require a patch version?

Reply:
This is a weird one. Good catch. Not sure how this crept in. :thinking: